host/crypto/rsa.h

```diff
@@ -11,4 +11,10 @@ oe_result_t oe_rsa_get_public_key_from_private(
     const oe_rsa_private_key_t* private_key,
     oe_rsa_public_key_t* public_key);
 
+/* Caller is responsible for freeing public key. */
+oe_result_t oe_rsa_get_public_key_from_engine(
```

Comment:
It is. I removed it.
At one time I was going to try to get the public key from the engine rather than private key, but that turned out not to be necessary as the private key from esrp contains adequate data to generate a public key.